src/App.tsx

```diff
@@ -1,223 +1,201 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import { useError } from './controllers/useError';
 import { AuthContext } from './components/Auth/AuthContext';
+import { ErrorNotification }
+  from './components/ErrorNotification/ErrorNotification';
+import { Footer } from './components/Footer/Footer';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { FilterType } from './types/FilterType';
+import { Todo } from './types/Todo';
+import { getCompletedTodoIds } from './components/helpers/helpers';
+import { todosApi } from './api/todos';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [completedFilter, setCompletedFilter] = useState(FilterType.All);
+  const [isAddingTodo, setIsAddingTodo] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [delitingTodoIds, setDeletingTodoIds] = useState<number[]>([]);
+  const [updatingTodoIds, setUpdatingTodoIds] = useState<number[]>([]);
+
+  const [showError, closeErroreMessage, errorMessages] = useError();
+
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
 
   useEffect(() => {
-    // focus the element with `ref={newTodoField}`
     if (newTodoField.current) {
       newTodoField.current.focus();
     }
-  }, []);
+
+    if (user) {
+      todosApi.getTodos(user.id)
+        .then(setTodos)
+        .catch(() => showError('Unable to load a todos'));
+    }
```

Comment:
you can use ENUM for errors
```suggestion
        .then(setTodos)
        .catch(() => showError(ErrorType.LoadTodo));
    }
```

Reply:
done